rust/persistent_storage/README.md

```diff
@@ -0,0 +1,202 @@
+# Persistent Storage
+The example dapp shows how to build a simple dapp in Rust, which will have persistent storage. The dapp is a very basic dapp with both backend and fronted, using Rust for the backend functionality and plain HTML and JavaScript for the frontend. The dapp is a simple counter, which will increment a counter by clicking a button in the frontend. 
+
+![Counter Frontend](README_images/frontend.png)
+
+## Introduction
+The purpuse of this example dapp is to build a minimalistic dapp, with persistent storage, based on the default dapp template installed by DFX when creating a new project. The dapp is a simple website with a counter. Every time a button is pressed, a counter is incremented. 
```

Comment:
Looks like similar copy/paste corrections required in this project, so I won't duplicate them here.
```suggestion
The purpose of this example dapp is to build a minimalistic dapp, with persistent storage, based on the default dapp template installed by DFX when creating a new project. The dapp is a simple website with a counter. Every time a button is pressed, a counter is incremented. 
```